warehouse/query-core/src/main/java/datawave/query/jexl/visitors/BoundedRangeDetectionVisitor.java

```diff
@@ -81,7 +85,32 @@ public Object visit(ASTNRNode node, Object data) {
         }
         
         return false;
-        
+    }
+    
+    // ensure we short circuit these nodes
+    @Override
+    public Object visit(ASTLTNode node, Object data) {
+        return data;
+    }
+    
+    @Override
+    public Object visit(ASTGTNode node, Object data) {
+        return data;
+    }
+    
+    @Override
+    public Object visit(ASTLENode node, Object data) {
+        return data;
+    }
+    
+    @Override
+    public Object visit(ASTGENode node, Object data) {
+        return data;
+    }
+    
+    @Override
```

Comment:
Add something like "Don't expect to see a bounded range inside of a function"

Reply:
Done